lib/initializer/parameter.rb

```diff
@@ -21,11 +21,9 @@ def definition
       end
     end
 
-    def self.build(name, visibility, default=nil)
+    def self.build(name, visibility, default=:no_default_value)
       instance = new(name, visibility)
 
-      default ||= :no_default_value
```

Comment:
What's the significance of moving the assignment of the default value to the method signature?